manticore/platforms/platform.py

```diff
@@ -1,10 +1,20 @@
+import wrapt
+import logging
 from ..utils.event import Eventful
 
+logger = logging.getLogger(__name__)
+
 
 class OSException(Exception):
     pass
 
 
+@wrapt.decorator
+def unimplemented(wrapped, _instance, args, kwargs):
+    logger.warning("Unimplemented system call: %s", wrapped.__name__)
```

Comment:
Is there any way to get the PC value here? So that it would print something like:
```
m.p.platform:WARNING: 0x402048: Unimplemented system call: sys_setsockopt
```

Another thing that might be useful is to display the args to the system call?
```diff
-     logger.warning("Unimplemented system call: %s", wrapped.__name__)
+     logger.warning("Unimplemented system call: %s -- args: %s", wrapped.__name__, args)
```
```
m.p.platform:WARNING: Unimplemented system call: sys_setsockopt -- args: (3, 1, 15, 140737488354584, 4)
```

Reply:
84afdf0 should be a little bit better. If I find the time, I'll work on modularizing the system call printing. I have some uncommitted changes I keep pushing between branches and it would be nice to be able to relegate those to a script.

Reply:
The change looks good! Thank you 👍